### ISSUE TITLE
Submit COPASI 4.30.233

### ISSUE THREAD

**Issue by jonrkarr** [biosimulators/Biosimulators#45]
## Instructions

Please use this issue to validate or contribute a containerized simulation tool, another version of the tool, or update a version of the tool. Specifically, edit the YAML syntax between the `---` below and then click the `Submit new issue` button.

Instructions for containerizing simulation tools and examples are available at https://biosimulators.org/help.

## Pre-submission checklist

**Is your containerized simulation tool compliant with BioSimulators' conventions?**

- [ ] The container provides a [BioSimulators-compliant command-line interface](https://biosimulators.org/help) that supports the [COMBINE/OMEX archive](https://combinearchive.org/) and [SED-ML](https://sed-ml.org) formats
- [ ] The `ENTRYPOINT` of the container is set to this command-line interface
- [ ] The specifications of the simulator follow the [BioSimulators format](https://api.biosimulators.org)
- [ ] Any additional ontology terms necessary to describe your simulator have been approved
      - [ ] Modeling frameworks: [SBO](https://sourceforge.net/p/sbo/term-request/)
      - [ ] Algorithms and their parameters: [KiSAO](https://sourceforge.net/p/kisao/feature-requests/new/)
      - [ ] Modeling formats: [EDAM](https://github.com/edamontology/edamontology/issues/new/choose)
      - [ ] Licenses: [SPDX](https://tools.spdx.org/app/submit_new_license/)
- [ ] Optional, the source code for the simulation tool is publicly available
- [ ] Optional, the simulation tool is available through a software repository such as CRAN or PyPI
- [ ] Optional, examples, tutorials, and documentation for the simulation tool are publicly available
- [ ] Optional, the simulation tool is registered with [bio.tools](https://bio.tools/)
- [ ] Optional, the container uses [labels to provide Open Containers Initiative-compliant metadata](https://biosimulators.org/help)
- [ ] Optional, the container uses [labels to provide BioContainers-compliant metadata](https://biosimulators.org/help)
- [ ] Optional, the container is registered with [BioContainers](https://biocontainers.pro/)

## Submission

**What is the id, version, and URL of the specifications of your simulation tool? Should BioSimulators validate your Docker image? Should BioSimulators save your simulator to the BioSimulators registry?**
Please follow the YAML template below.

---
id: copasi
version: 4.30.233
specificationsUrl: https://raw.githubusercontent.com/biosimulators/Biosimulators_COPASI/90f8d48f066dbbae367034824f39cc9cf38c12dd/biosimulators.json
validateImage: true
commitSimulator: true

---

## Post-submission approval workflow

Once you submit this issue, BioSimulators will use the following workflow to validate, approve, and optionally commit your simulator to the BioSimulators database:

1. An issue will be created with the label `Validate/submit simulator`.
2. An automated action will review your submission:
   1. Retrieve the specifications of your simulator.
   2. Validate these specifications.
   3. Optionally, validate the containerized simulator specified in your specifications.
3. If the validation fails, any errors will be posted to this issue. After fixing these issues, please edit the first block of this issue to re-initiate the automated validation.
4. If the validation succeeds, the `Validated` label will be added to the issue.
5. The submission of the first version of your simulator will be reviewed by the BioSimulators Team.
   1. If there are any concerns about your submission, the BioSimulators Team will use this issue to communicate them to you.
   2. The BioSimulators Team will work with you to resolve these concerns.
6. Once your simulator is approved, the BioSimulators Team will add the label `Approved` to this issue. 
7. If requested, the simulator will automatically be committed to the BioSimulators database and the issue will automatically be closed.

## Re-submission change list

If you need to revise your simulator to pass the validation, please use this section to briefly describe the changes that you made.

## Changes

- 1
- 2
- 3
- 4
- 5
- 6

**Comment by biosimulators-daemon** [comment]
Thank you @jonrkarr for your submission to the BioSimulators simulator validation/submission system!

The BioSimulators validator bot is [validating the specifications of your simulator, validating your Docker image and committing your simulator to the BioSimulators registry](https://github.com/biosimulators/Biosimulators/actions/runs/439008817). 

We will discuss any concerns with your submission in this issue.

**Comment by biosimulators-daemon** [comment]
The specifications of your simulator is valid!

**Comment by biosimulators-daemon** [comment]
Your simulator passed 3 test cases.

**Comment by biosimulators-daemon** [comment]
The image for your simulator is valid!

**Comment by biosimulators-daemon** [comment]
A member of the BioSimulators team will review your submission and publish your image before final committment to the registry.

**Comment by biosimulators-daemon** [comment]
Thank you @jonrkarr for your submission to the BioSimulators simulator validation/submission system!

The BioSimulators validator bot is [validating the specifications of your simulator, validating your Docker image and committing your simulator to the BioSimulators registry](https://github.com/biosimulators/Biosimulators/actions/runs/439014862). 

We will discuss any concerns with your submission in this issue.

**Comment by biosimulators-daemon** [comment]
The specifications of your simulator is valid!

**Comment by biosimulators-daemon** [comment]
Your simulator passed 3 test cases.

**Comment by biosimulators-daemon** [comment]
The image for your simulator is valid!

**Comment by biosimulators-daemon** [comment]
```diff
- The validation/submission of your simulator failed.
- 
-   500 Server Error for http+docker://localhost/v1.40/auth: Internal Server Error ("Get https://ghcr.io/v2/: denied")
- 
- Once you have fixed the problem, edit the first block of this issue to re-initiate this validation.
- 
- The BioSimulators Team is happy to help. Questions and feedback can be directed to the BioSimulators Team by posting comments to this issues that reference the GitHub team `@biosimulators/biosimulators` (without the backticks).
```

**Comment by biosimulators-daemon** [comment]
Thank you @jonrkarr for your submission to the BioSimulators simulator validation/submission system!

The BioSimulators validator bot is [validating the specifications of your simulator, validating your Docker image and committing your simulator to the BioSimulators registry](https://github.com/biosimulators/Biosimulators/actions/runs/439026657). 

We will discuss any concerns with your submission in this issue.

**Comment by biosimulators-daemon** [comment]
The specifications of your simulator is valid!

**Comment by biosimulators-daemon** [comment]
Your simulator passed 3 test cases.

**Comment by biosimulators-daemon** [comment]
The image for your simulator is valid!

**Comment by biosimulators-daemon** [comment]
```diff
- The validation/submission of your simulator failed.
- 
-   500 Server Error for http+docker://localhost/v1.40/auth: Internal Server Error ("Get https://ghcr.io/v2/: denied")
- 
- Once you have fixed the problem, edit the first block of this issue to re-initiate this validation.
- 
- The BioSimulators Team is happy to help. Questions and feedback can be directed to the BioSimulators Team by posting comments to this issues that reference the GitHub team `@biosimulators/biosimulators` (without the backticks).
```

**Comment by biosimulators-daemon** [comment]
Thank you @jonrkarr for your submission to the BioSimulators simulator validation/submission system!

The BioSimulators validator bot is [validating the specifications of your simulator, validating your Docker image and committing your simulator to the BioSimulators registry](https://github.com/biosimulators/Biosimulators/actions/runs/439097966). 

We will discuss any concerns with your submission in this issue.

**Comment by biosimulators-daemon** [comment]
The specifications of your simulator is valid!

**Comment by biosimulators-daemon** [comment]
Your simulator passed 3 test cases.

**Comment by biosimulators-daemon** [comment]
The image for your simulator is valid!

**Comment by biosimulators-daemon** [comment]
```diff
- The validation/submission of your simulator failed.
- 
-   500 Server Error for http+docker://localhost/v1.40/auth: Internal Server Error ("Get https://ghcr.io/v2/: denied")
- 
- Once you have fixed the problem, edit the first block of this issue to re-initiate this validation.
- 
- The BioSimulators Team is happy to help. Questions and feedback can be directed to the BioSimulators Team by posting comments to this issues that reference the GitHub team `@biosimulators/biosimulators` (without the backticks).
```

**Comment by biosimulators-daemon** [comment]
Thank you @jonrkarr for your submission to the BioSimulators simulator validation/submission system!

The BioSimulators validator bot is [validating the specifications of your simulator, validating your Docker image and committing your simulator to the BioSimulators registry](https://github.com/biosimulators/Biosimulators/actions/runs/439123614). 

We will discuss any concerns with your submission in this issue.

**Comment by biosimulators-daemon** [comment]
The specifications of your simulator is valid!

**Comment by biosimulators-daemon** [comment]
Your simulator passed 3 test cases.

**Comment by biosimulators-daemon** [comment]
The image for your simulator is valid!

**Comment by biosimulators-daemon** [comment]
```diff
- The validation/submission of your simulator failed.
- 
-   500 Server Error for http+docker://localhost/v1.40/auth: Internal Server Error ("Get https://ghcr.io/v2/: denied")
- 
- Once you have fixed the problem, edit the first block of this issue to re-initiate this validation.
- 
- The BioSimulators Team is happy to help. Questions and feedback can be directed to the BioSimulators Team by posting comments to this issues that reference the GitHub team `@biosimulators/biosimulators` (without the backticks).
```

**Comment by biosimulators-daemon** [comment]
Thank you @jonrkarr for your submission to the BioSimulators simulator validation/submission system!

The BioSimulators validator bot is [validating the specifications of your simulator, validating your Docker image and committing your simulator to the BioSimulators registry](https://github.com/biosimulators/Biosimulators/actions/runs/439143404). 

We will discuss any concerns with your submission in this issue.

**Comment by biosimulators-daemon** [comment]
The specifications of your simulator is valid!

**Comment by biosimulators-daemon** [comment]
Your simulator passed 3 test cases.

**Comment by biosimulators-daemon** [comment]
The image for your simulator is valid!

**Comment by biosimulators-daemon** [comment]
```diff
- The validation/submission of your simulator failed.
- 
-   Unable to push image to ghcr.io/biosimulators/copasi:4.30.233
- 
- Once you have fixed the problem, edit the first block of this issue to re-initiate this validation.
- 
- The BioSimulators Team is happy to help. Questions and feedback can be directed to the BioSimulators Team by posting comments to this issues that reference the GitHub team `@biosimulators/biosimulators` (without the backticks).
```

**Comment by biosimulators-daemon** [comment]
Thank you @jonrkarr for your submission to the BioSimulators simulator validation/submission system!

The BioSimulators validator bot is [validating the specifications of your simulator, validating your Docker image and committing your simulator to the BioSimulators registry](https://github.com/biosimulators/Biosimulators/actions/runs/439168919). 

We will discuss any concerns with your submission in this issue.

**Comment by biosimulators-daemon** [comment]
The specifications of your simulator is valid!

**Comment by biosimulators-daemon** [comment]
Your simulator passed 3 test cases.

**Comment by biosimulators-daemon** [comment]
The image for your simulator is valid!

**Comment by biosimulators-daemon** [comment]
Your submission was committed to the BioSimulators registry. Thank you!

Future submissions of subsequent versions of copasi to the BioSimulators registry will be automatically validated. These submissions will not require manual review by the BioSimulators Team.

**Comment by biosimulators-daemon** [comment]
Thank you @jonrkarr for your submission to the BioSimulators simulator validation/submission system!

The BioSimulators validator bot is [validating the specifications of your simulator, validating your Docker image and committing your simulator to the BioSimulators registry](https://github.com/biosimulators/Biosimulators/actions/runs/439188867). 

We will discuss any concerns with your submission in this issue.

**Comment by biosimulators-daemon** [comment]
The specifications of your simulator is valid!

**Comment by biosimulators-daemon** [comment]
Your simulator passed 3 test cases.

**Comment by biosimulators-daemon** [comment]
The image for your simulator is valid!

**Comment by biosimulators-daemon** [comment]
Your submission was committed to the BioSimulators registry. Thank you!

Future submissions of subsequent versions of copasi to the BioSimulators registry will be automatically validated. These submissions will not require manual review by the BioSimulators Team.
